### ISSUE TITLE
Error when using RedOx together with JDT and Enhanced Class Decompiler

### ISSUE THREAD

**Issue by fwcd** [LucasBullen/redOx#25]
Eclipse version: Oxygen.2 (4.7.2)
RedOx version: 0.1.0.201801181621
Enhanced Class Decompiler version: 3.0.0.201709092115

Every time I open a decompiled (Java) ".class" file using the "Enhanced Class Decompiler" the following popup shows up:

```
An error has occurred. See error log for more details.
org.eclipse.jdt.internal.ui.javaeditor.InternalClassFileEditorInput cannot be cast to org.eclipse.ui.part.FileEditorInput
```

Strangely enough, this error occurs when neither the Rust-perspective is selected, nor any RedOx-specific view is open and can easily be reproduced. It disappears after uninstalling RedOx.

**Comment by LucasBullen** [comment]
@TheDroide would you be able to review that the [pull request #26](https://github.com/LucasBullen/redOx/pull/26) solves the issue?

If so, follow the [git installation instructions](https://github.com/LucasBullen/redOx/blob/master/documentation/Installation.md#using-github) and checkout the PR branch: `git checkout -b jdtcasting origin/jdtcasting`

If not, also fine but please comment such so that I am not waiting.

**Comment by TheDroide** [comment]
I will review it as soon as I have the time to do it.

**Comment by LucasBullen** [comment]
I have merged the PR to get the fix out for others and yourself sooner (f40c429), if you are to find issues with the patch, reopen this issue and we will address them.
